spec/factories.rb

```diff
@@ -9,4 +9,47 @@
     password { 'password' }
     password_confirmation { 'password' }
   end
+
+  factory :merchant do
+    name { Faker::TvShows::HeyArnold.location }
+    address { Faker::Address.street_address }
+    city { Faker::Address.city }
+    state { Faker::Address.state_abbr }
+    zip { Faker::Address.zip }
+    
+    trait :with_items do
+      transient do 
+        item_count { 3 }
+      end
+      
+      after(:create) do |merchant, evaluator|
+        merchant.items << create_list(:item, evaluator.item_count)
+      end
+    end
+  end
+
+  factory :item do
+    sequence(:name) { |n| "Item ##{n}" }
+    description { Faker::Lorem.sentence }
+    price { Faker::Commerce.price }
+    image { 'https://semantic-ui.com/images/wireframe/image.png' }
+    inventory { Faker::Number.between(from: 12, to: 34) }
+
+    merchant
+  end
+
+  factory :order do
+    name { Faker::Name.name }
+    address { Faker::Address.street_address }
+    city { Faker::Address.city }
+    state { Faker::Address.state_abbr }
+    zip { Faker::Address.zip }
```

Comment:
Are there tests to this? If not, can we add a field for a user_id to the order factory? It'll make sense with my PR (orders must have a user_id) and what Shaunda will push soon.

Reply:
Agree this would be helpful!

Reply:
So if you look at the schema the orders table doesn't actually have a user_id column yet so you would need to run a migration before creating a new factory bot field for that.

Do you want me to run that migration or do you want to wait?

Reply:
Ah, what we mean is ours (so main) has it right now (the migration files). If you can't do it before you do the migration, we can wait for it. No worries!

Reply:
Oh ok got it! Yes I can go ahead and add that if you want, but it will break all of my tests until this is merged in (which is fine with me). I wont be able to create an order if I add a user_id to the factory since it looks at the model and won't know what that is